app/controllers/clients/cas_readiness_controller.rb

```diff
@@ -14,7 +14,7 @@ class CasReadinessController < ApplicationController
     after_action :log_client
 
     def edit
-      @neighborhoods = Cas::Neighborhood.order(:name).pluck(:id, :name) if Cas::Neighborhood.db_exists?
+      @neighborhoods = CasAccess::Neighborhood.order(:name).pluck(:id, :name) if CasAccess::Neighborhood.db_exists?
```

Comment:
I'm slightly uncomfortable with the core app depending on driver functionality... But, I suppose this is a stopgap to moving interactions with CAS out of the core anyway...

Reply:
Yeah, I think that's a TODO.  There are still quite a few places where the CAS stuff is getting called directly, and now that's dependent on the driver (since the base classes all moved there.)  I think phase 2 should be to move anything strictly CAS related into the driver, and phase 3 to inject into the main app instead of spreading it around.